frontend/src/components/sidebar/index.tsx

```diff
@@ -2,15 +2,22 @@ import React from 'react';
 import sidebarStyles from './sidebar.module.scss';
 import sidebarOptions, { SidebarOption } from '../../constants/sidebarOptions';
 
-const getButtons = (options: SidebarOption[]) =>
+const getButtons = (options: SidebarOption[], selctedOption: String) =>
   options.map((o) => (
-    <button type="button" className={sidebarStyles.menuOption} key={o.name}>
+    <button
+      type="button"
+      className={`${sidebarStyles.menuOption} ${
+        o.name === selctedOption ? sidebarStyles.selectedMenuOption : ''
```

Comment:
If the second part of a statement like this is null, or an empty string, you could also write it like `${o.name === selectedOption && sidebarStyles.selectedMenuOption}`

Reply:
The html changes to this ```<button type="button" class="sidebar_menuOption__12WdO false">Rota</button>```   We have a false in the class. I think the reason this doesnt work is what you are suggesting is a JSX thing but doesnta pply to JS logic in strings?

Reply:
Yeah I think you're right. That's annoying